bokehjs/src/lib/models/glyphs/glyph.ts

```diff
@@ -306,6 +307,7 @@ export abstract class GlyphView extends View {
     }
 
     this._map_data()
+    this.glglyph?.set_data_changed()
```

Comment:
This is necessary since we are no longer mapping data coordinates in gl, we need to pick up the new screen mapped coordinates.